api/v1/observability_types.go

```diff
@@ -87,6 +87,7 @@ type SelfContained struct {
 	PrometheusOperatorResourceRequirement v1.ResourceRequirements  `json:"prometheusOperatorResourceRequirement,omitempty"`
 	GrafanaResourceRequirement            *v1.ResourceRequirements `json:"grafanaResourceRequirement,omitempty"`
 	GrafanaOperatorResourceRequirement    v1.ResourceRequirements  `json:"grafanaOperatorResourceRequirement,omitempty"`
+	GrafanaVersion                        string                   `json:"grafanaVersion,omitempty"`
```

Comment:
i dont think this is needed, thats for the next step of making it self contained for projects that dont use config repos i.e RHOAM